html/views/overlay/index.js

```diff
@@ -178,12 +188,13 @@ function ensureSkaterExists(skaterId, team) {
 	}
 }
 
-function updateSkaterNumber(me,mb,v) {
-	sv = v;
-	if(v == '' || v == '-' || v == null) { sv = 'ZZZB'; v = '-'; }
```

Comment:
Again, method parameter v here is assigned to but not referenced later so I believe this is extraneous.